test/runtests.jl

```diff
@@ -128,20 +128,20 @@ get_fn_body(ex) =
     scope = first_scope_except_top(a)[1]
     @test length(scope.freevars) == 2
 
-    a = solve(:(function z(x, k=1)
+    a = solve!(:(function z(x, k=1)
                    (k=k, )
                end
     ))
 
     @test any(x -> x.name == :k, get_fn_scope(a).bounds)
 
-    @test @when :(k=$_, ) = get_fn_body(a).args[2] begin
+    @test @when :(k=$_, ) = get_fn_body(a).args[end] begin
```

Comment:
I needed to change `.args[2]` to `.args[end]` to make it work with Julia 1.5-DEV.  It seems the AST was changed a bit in Julia 1.5-DEV:

```julia
julia> :(function f()
           a
       end)
:(function f()
      #= REPL[1]:1 =#
      #= REPL[1]:2 =#
      a
  end)
```

while it is

```julia
julia> :(function f()
           a
       end)
:(function f()
      #= REPL[1]:2 =#
      a
  end)
```

in Julia 1.4 and before.